Source/Utility/Utilities/UnitConversions.H

```diff
@@ -0,0 +1,241 @@
+#ifndef UNITCONVERSIONS_H
+#define UNITCONVERSIONS_H
+
+#include <AMReX.H>
+#include <AMReX_REAL.H>
+#include <AMReX_Utility.H>
+
+namespace pele::physics::utilities {
+
+struct units
+{
+  // CGS to MKS conversions
+
+  AMREX_GPU_HOST_DEVICE
+  AMREX_FORCE_INLINE
+  amrex::Real CGS2MKS_LENGTH(const amrex::Real L_cgs)
+  {
+    return L_cgs * 1.0e-2; // cm to m
+  }
+
+  AMREX_GPU_HOST_DEVICE
+  AMREX_FORCE_INLINE
+  amrex::Real CGS2MKS_LENGTH_POW(const amrex::Real L_cgs, amrex::Real n)
```

Comment:
const the second arg? Same for a bunch of others in this file.

Reply:
Ah yes, good catch.